test/server/server_corpus/clusterfuzz-testcase-minimized-server_fuzz_test-5686444035670016

```diff
@@ -0,0 +1,2 @@
+static_resources {   clusters {     name: " "     connect_timeout {       nanos: 4     }     hosts {
```

Comment:
Could you also add a unit test for this new clamping or conversion boundary behavior somewhere, e.g. https://github.com/envoyproxy/envoy/blob/master/test/common/event/dispatcher_impl_test.cc#L189? The corpus entries are useful, but they are hard to reason about when trying to understand behavior, and are subject to fuzzer rot.

Reply:
I added some tests. I refactored the timeval conversion to a separate function to make testing easier.